ramls/oai-pmh-view.raml

```diff
@@ -49,4 +51,53 @@ resourceTypes:
           type: boolean
           required: false
           default: true
-
+  /updatedInstanceIds:
+    displayName: Stream API to get instances ids for OAI-PMH
+    type:
+      collection-get:
+       schemaCollection: oaipmhUpdatedInstanceIds
+       exampleCollection: !include examples/oaipmhinstanceids.json
+    get:
+      description: Stream updated instances ids for oai-pmh
+      is:
+        [validate]
+      queryParameters:
+        startDate:
+          description: start date
+          type: string
+          required: false
+        endDate:
+          description: start date
+          type: string
+          required: false
+        deletedRecordSupport:
+          description: return deleted records
+          type: boolean
+          required: false
+          default: true
+        skipSuppressedFromDiscoveryRecords:
+          description: skip discovery suppressed records
+          type: boolean
+          required: false
+          default: true
+  /enrichedInstances:
+    displayName: Stream API to get instances with items and holdings for OAI-PMH
+    post:
+      description: Stream instances view data for oai-pmh
+      is:
+        [validate]
+      body:
+        application/json:
+          description: List of instances Ids
+          type: string[]
```

Comment:
Usually a schema is used to define a JSON request or response. What does `string[]` mean in the context of a JSON response?

Reply:
I introduced a new type for POST payload which carry bith "instance ids" and "skipSuppressedFromDiscovery" data. So now there is only payload passed with parameters.